client/api/index.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+const url = 'https://ciberpizza.herokuapp.com/api/products'

```

Comment:
@AStashevskaya Its your `deployed` app url, but for development purpose we should target requests to our runned development server, I think you could try

```js
const url = '/api/products'
```